package.json

```diff
@@ -52,7 +52,7 @@
     "less-plugin-autoprefix": "1.5.1",
     "less-plugin-clean-css": "1.5.1",
     "nomnom": "1.8.1",
-    "openlayers": "fredj/ol3#afb2084",
+    "openlayers": "fredj/ol3#b49546d",
```

Comment:
That branch still includes the change in ol3 that removed the provide for `ol.style.StyleFunction`, right?
See https://github.com/camptocamp/ngeo/pull/1180#issuecomment-218204325